src/pages/gateway/create-mesh.md

```diff
@@ -80,6 +80,27 @@ When creating or updating a mesh, the file to upload must have the `.json` filen
 
 Each workspace within a project can only have one mesh associated with it at a time.
 
+## Mesh example
+
+The following mesh file can be used with minimal setup to access the endpoint of the [Venia] demo store.
+
+``` json
+{
+"meshConfig": {
+    "sources": [
+      {
+        "name": "Commerce",
+        "handler": {
+          "graphql": {
+            "endpoint": "https://venia.magento.com/graphql/"
+          }
+        }
+      }
+    ]
+  }
+}
+```
+
```

Comment:
For Accessing the gateway, are we documenting the steps for logging into the dev console and creating a key? cc: @revanth0212

Reply:
Since this is a developer guide folks looking at it are most like developers. They need a way to create API keys to access the mesh and I'd assume they will have access to the developer console. I don't see a need to call it out for customers.